packages/next/client/page-loader.js

```diff
@@ -182,6 +195,15 @@ export default class PageLoader {
 
   async prefetch(route, isDependency) {
     route = this.normalizeRoute(route)
+
+    if (
+      /\.css$/.test(route) &&
+      !document.querySelector(`link[href^="${route}"]`)
+    ) {
+      this.loadStylesheet(route)
```

Comment:
Prefetching should be done via a real prefetch, not via adding the style tag!
i.e.
```html
<link rel="prefetch" href="/style.css" as="style" />
```

Reply:
Hmm, I've tried both `preload` and `prefetch`, but the browser (Safari) doesn't work as expected (checked preload/prefetch support before), so the only way to "preload" the CSS files would be the current `loadStylesheet` method.
Alternatively, we could just return blank on prefetching CSS and simply load it on page transition which seems to work without any FOUC or sth.